README.md

```diff
@@ -1,25 +1,25 @@
-Every day, millions of people come up with ideas that have the potential to change the world. Yet almost all ideas do not take any form mainly because of lack of support from peers with similar vision or the originator does not have a clear path of how to make it a reality. 
+Every day millions of people come up with ideas that have the potential to change the world. Yet almost all ideas do not take any form, mainly because of lack of support from peers with similar vision, or the originator does not have a clear path of how to make it a reality. 
 
 > ## We exist to take your ideas to execution. 
 
-We are different from any freelancing program as in the process, you learn new skills and thousands of passionate people are beside your shoulder for the entire journey. You do not owe a penny to us. 
+We are different from any freelancing program as in the process you learn new skills and thousands of passionate people are beside your shoulder for the entire journey. You do not owe a penny to us. 
 
-> We do everything to bring a smile on your face. 😄 
+> We do everything to bring a smile to your face. 😄 
 
 This is how it works: 
 
 ![alt text](https://github.com/AdiChat/Freelance/blob/master/data/Promo.jpg "OpenGenus Freelance")
 
 * Join us at OpenGenus by opening an issue [here](https://github.com/OpenGenus/Join_OpenGenus)
-* You will be welcomed to our community in a day. 
-* Once you join, create an issue or get in touch at **Slack**. Tell us your vision and what you want to do. 
-* We will direct you to **free** resources on the web to attain essential skills to make your vision a reality. We will, also, guide you through the entire course. We will try to keep this period short as much as a month. 
+* You will be welcomed intto our community within a day. 
+* Once you join, create an issue or get in touch on **Slack**. Tell us your vision and what you want to do. 
```

Comment:
_at_ was correct. However, I would change to: ... get in touch _with us at_ Slack.